include/ttmlir/Conversion/Passes.h

```diff
@@ -7,6 +7,7 @@
 
 #ifdef TTMLIR_ENABLE_STABLEHLO
 #include "ttmlir/Conversion/ArithToStableHLO/ArithToStableHLO.h"
+#include "ttmlir/Conversion/RedundantBroadcastElimination/RedundantBroadcastElimination.h"
```

Comment:
I think, we should make it a generic pass instead of StableHLO dependent pass. Other dialects may also require this optimization.

Reply:
We don't know yet if other front-ends will also require it, so far it seems to be a stablehlo issue only, we can replicate to other front-ends if needed in future.

Reply:
I agree with @mmanzoorTT on this. I don't want to block the PR on getting the other pipeline in place, but pure TTIR transformations should not happen in the stablehlo to TTIR pipeline.